rskj-core/src/main/java/org/ethereum/config/blockchain/upgrades/NetworkUpgrade.java

```diff
@@ -28,7 +28,8 @@ public enum NetworkUpgrade {
     PAPYRUS_200("papyrus200"),
     TWOTOTHREE("twoToThree"),
     IRIS300("iris300"),
-    HOP400("hop400");
+    HOP400("hop400"),
+    RSKIP326("rskip326");
```

Comment:
We don't have a codename defined for 5.0 yet, so let's reuse `hop`
```suggestion
    HOP500("hop500");
```

Reply:
Updated!